pkg/interactors/register_thing.go

```diff
@@ -0,0 +1,124 @@
+package interactors
+
+import (
+	"strconv"
+
+	"github.com/CESARBR/knot-babeltower/pkg/logging"
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+)
+
+// RegisterThing use case to register a new thing
+type RegisterThing struct {
+	logger       logging.Logger
+	msgPublisher network.Publisher
```

Comment:
What do you think about rename to `publish`?

Reply:
For the name, we need the think about the data.publish messages too. Maybe publish is not a good name for this case.

Reply:
IMO, we can approve this PR as it is and change this name when adding the publish data operation. What do you think, @vitbaq??

Reply:
I agree with @joaoaneto